core/commands/files/files.go

```diff
@@ -694,52 +716,52 @@ Examples:
 `,
 	},
 
-	Arguments: []cmds.Argument{
-		cmds.StringArg("path", true, false, "Path to dir to make."),
+	Arguments: []cmdsutil.Argument{
+		cmdsutil.StringArg("path", true, false, "Path to dir to make."),
 	},
-	Options: []cmds.Option{
-		cmds.BoolOption("parents", "p", "No error if existing, make parent directories as needed."),
+	Options: []cmdsutil.Option{
+		cmdsutil.BoolOption("parents", "p", "No error if existing, make parent directories as needed."),
 	},
 	Run: func(req cmds.Request, res cmds.Response) {
 		n, err := req.InvocContext().GetNode()
 		if err != nil {
-			res.SetError(err, cmds.ErrNormal)
+			res.SetError(err, cmdsutil.ErrNormal)
 			return
 		}
 
 		dashp, _, _ := req.Option("parents").Bool()
 		dirtomake, err := checkPath(req.Arguments()[0])
 		if err != nil {
-			res.SetError(err, cmds.ErrNormal)
+			res.SetError(err, cmdsutil.ErrNormal)
 			return
 		}
 
 		flush, _, _ := req.Option("flush").Bool()
 
 		err = mfs.Mkdir(n.FilesRoot, dirtomake, dashp, flush)
 		if err != nil {
-			res.SetError(err, cmds.ErrNormal)
+			res.SetError(err, cmdsutil.ErrNormal)
 			return
 		}
-
+		res.SetOutput(nil)
```

Comment:
Yes, this is only for legacy `Run` functions. Also realized I can't simply call `SetOuptut` after `Run`, because some `Run`s call it in a goroutine and we are waiting for that. I'll just keep it the way it is.